.travis.yml

```diff
@@ -37,6 +37,12 @@ matrix:
     - php: 7.0
       env: PHPCS=1 DEFAULT=0
 
+    - php: 7.0
```

Comment:
I am just a bit worried about the average build time then as it will further increase with each specific run. But I guess for the time being it doesnt hurt too much and those phpstan builds are probably rather fast.